PWGGA/GammaConv/AliDalitzData.cxx

```diff
@@ -32,16 +32,18 @@ ClassImp( AliDalitzData )
     
         AliDalitzData::~AliDalitzData(){
         }
-    
-     AliDalitzAODESD* AliDalitzData::GetTrack(Int_t i){
+
+    auto_ptr<AliDalitzAODESD> AliDalitzData::GetTrack(Int_t i){
```

Comment:
Hello Andrew,
reading and coding, unique_ptr is not the same that auto_ptr, not event shared_ptr, shared and unique they are very well defined in terms on where they live and the pointers, meanwhile, auto_ptr is more flexible in that way, for not said bad defined. For example here, on this part I define this switch tracks for AOD and ESD with auto, changing with unique didnt, permit to keep using the Particle the same way.
`    unique_ptr<AliDalitzAODESD> AliDalitzData::GetTrack(Int_t i){
        
         if (fIsESD==kTRUE){
                unique_ptr<AliDalitzAODESD> esdtrack( new AliDalitzAODESD((AliESDtrack*)fESDEvent->GetTrack(i)));
                esdtrack->ComputeImpactParameter();
                return esdtrack;}
         else { unique_ptr<AliDalitzAODESD> aodtrack( new AliDalitzAODESD((AliAODTrack*)fAODEvent->GetTrack(i)));
                aodtrack->ComputeImpactParameter(fAODEvent->GetPrimaryVertex(),fAODEvent->GetMagneticField());
                return aodtrack;}
     } `